### DISCUSSION TITLE
How to become a contributor to UCX

### DISCUSSION THREAD

**Discussion by rkkalluri-dbx** [databrickslabs/ucx#1563] [General]
What are some easy tickets we can pick as a new contributor to UCX. Is there a document on setting up your dev environment to develop for UCX

**Comment by nfx** [comment]
Easy tickets are marked as good first issue. And there's contributing.md

**Comment by rkkalluri-dbx** [comment]
Is there a slack channel or other means to interact with the team or is it mostly email and issues.

From: Serge Smertin ***@***.***>
Date: Friday, April 26, 2024 at 11:54 AM
To: databrickslabs/ucx ***@***.***>
Cc: Rajesh Kalluri ***@***.***>, Author ***@***.***>
Subject: Re: [databrickslabs/ucx] How to become a contributor to UCX (Discussion #1563)

Easy tickets are marked as good first issue. And there's contributing.md

**Reply by nfx** [reply]
@rkkalluri-dbx It's mainly through issues and PRs. Active contributors have different rules
